src/main/kotlin/no/nav/syfo/cronjob/journalpostdistribusjon/DialogmoteJournalpostDistribusjonCronjob.kt

```diff
@@ -24,6 +24,7 @@ class DialogmoteJournalpostDistribusjonCronjob(
         referatJournalpostDistribusjon()
     }
 
+    // TODO: Kan dette slettes?
```

Comment:
Dette er for å sende fysiske brev til arbeidstakere som har reservert seg mot digital kommunikasjon. Har dere løst det i esyfo?

Reply:
Nåværende utsending av fysisk brev må omgjøres litt. Tenker at jeg lager egen PR for det(slette cron jobb og lagring av brev+pdf i databasee), sånn at denne PRen ikke blir for stor.